lib/solidus_subscriptions/configuration.rb

```diff
@@ -11,7 +11,7 @@ class Configuration
       :success_dispatcher_class, :failure_dispatcher_class, :payment_failed_dispatcher_class,
       :out_of_stock_dispatcher, :maximum_successive_skips, :reprocessing_interval,
       :minimum_cancellation_notice, :processing_queue, :subscription_line_item_attributes,
-      :subscription_attributes, :subscribable_class,
+      :subscription_attributes, :subscribable_class, :process_job_error_handler,
```

Comment:
Can we call this `processing_error_handler`, to future-proof it in case the processing is moved to some other place in the future?